vm.c

```diff
@@ -50,15 +67,15 @@
 /* OPCODE impl max size */
 #define OPCODE_IMPL_MAX_SIZE 256
 
-typedef struct __vm_env {
+struct __vm_env {
```

Comment:
Yes, it should. The main reason to separate is because your PR is focus on adding new J-series operator, not "Fix duplicate typedef". 

When in future someone trying to investigate in git log, it will be a mess if irrelevant changed is in the commit.